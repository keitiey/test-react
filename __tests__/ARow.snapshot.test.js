import renderer from 'react-test-renderer';
import React from 'react';
import ARow from '../src/components/content/table/row/row';

describe('snapshots ARow component', () => {
  test('ARow component renders the company correctly', () => { //eslint-disable-line
    const company = {
      code: 'ANZ.AX',
      name: 'AUSTRALIA AND NEW ZEALAND BANKING GROUP LIMITED',
      price: 29.17,
      volume: 673111,
      value: 19634648,
      change: 0.39,
      percentChange: 1.36
    };
    const rendered = renderer.create(
      <ARow company={company} />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
