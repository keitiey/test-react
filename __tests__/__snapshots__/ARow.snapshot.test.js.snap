// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ARow component renders the company correctly 1`] = `
<tr
  className={undefined}
  style={
    Object {
      "backgroundColor": "#FDFDFD",
      "borderBottom": "1px solid #e0e0e0",
      "color": "rgba(0, 0, 0, 0.87)",
      "height": 48,
      "muiPrepared": true,
      "tableRow": Object {
        "borderBottomWidth": "2px",
      },
    }
  }
>
  <td
    className={undefined}
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "inherit",
        "color": "#529AE2",
        "fontSize": 13,
        "fontWeight": "bolder",
        "height": 48,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingLeft": 24,
        "paddingRight": 24,
        "textAlign": "left",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": "10%",
      }
    }
  >
    ANZ.AX
  </td>
  <td
    className={undefined}
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "inherit",
        "color": "#777777",
        "fontSize": 13,
        "fontWeight": "500",
        "height": 48,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingLeft": 24,
        "paddingRight": 24,
        "textAlign": "left",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": "40%",
      }
    }
  >
    AUSTRALIA AND NEW ZEALAND BANKING GROUP LIMITED
  </td>
  <td
    className={undefined}
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "inherit",
        "fontSize": 13,
        "fontWeight": "bolder",
        "height": 48,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingLeft": 24,
        "paddingRight": 24,
        "textAlign": "right",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": "10%",
      }
    }
  >
    29.17
  </td>
  <td
    className={undefined}
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "inherit",
        "display": "none",
        "fontSize": 13,
        "height": 48,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingLeft": 24,
        "paddingRight": 24,
        "textAlign": "left",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
      }
    }
  >
    673111
  </td>
  <td
    className={undefined}
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "inherit",
        "fontSize": 13,
        "fontWeight": "bold",
        "height": 48,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingLeft": 24,
        "paddingRight": 24,
        "textAlign": "right",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": "20%",
      }
    }
  >
    19,634,648
  </td>
  <td
    className={undefined}
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "inherit",
        "color": "#08D66B",
        "fontSize": 13,
        "fontWeight": "bold",
        "height": 48,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingLeft": 24,
        "paddingRight": 24,
        "textAlign": "right",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": "10%",
      }
    }
  >
    0.39
  </td>
  <td
    className={undefined}
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "backgroundColor": "inherit",
        "color": "#08D66B",
        "fontSize": 13,
        "fontWeight": "bold",
        "height": 48,
        "muiPrepared": true,
        "overflow": "hidden",
        "paddingLeft": 24,
        "paddingRight": 24,
        "textAlign": "right",
        "textOverflow": "ellipsis",
        "whiteSpace": "nowrap",
        "width": "10%",
      }
    }
  >
    1.36
    %
  </td>
</tr>
`;
